src/app/globals.css

```diff
@@ -7,3 +7,8 @@ body {
   background-color: '#FEFDFD';
   color: black;
 }
+
+.custom-footer::before {
```

Comment:
**:hammer: Ajuste**

No código, você tá definindo o z index direto com as classes do tailwind sem usar essa classe "custom-footer"! 

Se não for utilizar, é bom remover do código (ps: se for utilizar, pode deixar! é que não está sendo chamado em lugar nenhum)